api/models/site.js

```diff
@@ -34,6 +44,22 @@ const beforeValidate = (site) => {
   }
 };
 
+
+function isEmptyOrUrl(value, throws = true) {
+  const validUrlOptions = {
+    require_protocol: true,
+    protocols: ['http', 'https'],
```

Comment:
Definitely scope creep for the PR, but definitely a good idea, since Federalist already gives free HTTPS to all customers. 😺